### ISSUE TITLE
Icon Rendering Issues

### ISSUE THREAD

**Issue by cryofault** [EliverLara/AndromedaLauncher#2]
I've been using your launcher but I'm not able to figure out what's causing this issue with the menu icons. My first thoughts are that this is something on my end but I'm running an RX 580 with the included Mesa drivers + a clean Plasma 6 install and these icons seem to be the only ones with any trouble. Any ideas?

![Screenshot_20240406_042500](https://github.com/EliverLara/AndromedaLauncher/assets/25939455/ba531944-d229-4535-afcb-08d89c4b782d)
![Screenshot_20240406_042511](https://github.com/EliverLara/AndromedaLauncher/assets/25939455/4ccc606c-cb46-423d-836b-079581aebf2e)
![Screenshot_20240406_042656](https://github.com/EliverLara/AndromedaLauncher/assets/25939455/a9aa4c13-5053-4007-8bf4-e3877b59667a)

**Comment by tjumar** [comment]
I'm having the same issues with on arch, rx6700 gpu, plasma 6, wayland.
Journal shows: "Texture used with different accesses within the same pass, this is not allowed."
Googling the error it seems an issue with ColorOverlay from Qt5Compat.GraphicalEffects (extra/qt6-5compat is installed). 
A quick and dirty commenting out all of ColorOverlay { } occurrences seems to remedy the issue.

**Comment by ElijahHW** [comment]
Having the exact same issues here too.

**Comment by MattyGWS** [comment]
Same issues here on Fedora. Wayland, AMD 6950xt GPU, AMD 5950x CPU.

**Comment by Bilalharoon** [comment]
I had the Same issue. I removed all the overlays in Main.qml and Headers.qml in /home/bilalharoon/.local/share/plasma/plasmoids/AndromedaLauncher/contents/ui

```js
  ColorOverlay {
      visible: true
       anchors.fill: headerLabel
       source: headerLabel
       color: main.textColor
     }
```
I don't know why they are there. I will try to submit a pr to remove them

**Comment by Bilalharoon** [comment]
https://github.com/EliverLara/AndromedaLauncher/pull/8[https://github.com/EliverLara/AndromedaLauncher/pull/8]

**Comment by PLAYERUNKNOWNHatsune01** [comment]
Me too here. Do we have any solution or updates to fix the problem?

**Comment by Bilalharoon** [comment]
> Me too here. Do we have any solution or updates to fix the problem?

Install my fork for now

**Comment by EliverLara** [comment]
Hi I'm sorry for the very very late response(github never notified me anything about this repo) but, is anyone still facing this issue?

**Comment by omuas** [comment]
yes

**Comment by Leniwcowaty** [comment]
![Image](https://github.com/user-attachments/assets/25713d57-eb75-4333-9c8d-ba32e7cdc144)
Can confirm - still issues with rendering the icons... Plasma 6.2.5 on Fedora 41 (Wayland)

**Comment by EliverLara** [comment]
@Leniwcowaty since I'm not able to reproduce the issue, Could you please help me to test if replacing the` Header.qml` file located in `~/.local/share/plasma/plasmoids/AndromedaLauncher/contents/ui/` with [this one](https://github.com/user-attachments/files/18646762/Header.zip) fixes the issue in the top buttons?

If you do it please make sure to reload plasma shell by opening **krunner** and executing `plasmashell --replace`

**Comment by Leniwcowaty** [comment]
@EliverLara sure, when this bug occurs again (I also cannot consistently reproduce it, it just happens after a bit of time and several instances of launching Andromeda) I will try this and come back to you

**Comment by Leniwcowaty** [comment]
@EliverLara yes, that fixed the Settings and Shutdown icons, favourites and search icons are still borked

**Comment by omuas** [comment]
@EliverLara @Leniwcowaty Can confirm, the provided Header.qml fixes Settings and Shutdown Icons.

**Comment by EliverLara** [comment]
Thanks for your help! I've fixed the issues and published a new version on [kde-store](https://www.pling.com/p/2144212/)

**Comment by Leniwcowaty** [comment]
Can confirm - the issue no longer happens on all three of my machines
